{
  "name": "Unreachable 'except' block",
  "description": "Handling general exceptions before specific exceptions means that the specific",
  "cwe": "quality",
  "cwe-description": null,
  "language": "python",
  "checker-language": "ql",
  "example": null,
  "loc": 17,
  "test": [],
  "branches": 2,
  "apis": 13
}